server/player/player.go

```diff
@@ -1079,6 +1117,18 @@ func (p *Player) StopFlying() {
 	p.session().SendGameMode(p.GameMode())
 }
 
+// Blocking returns true if the player is currently blocking with a shield. The first boolean is true if the player was
+// holding a shield. The second boolean is true if the player was performing the necessary actions in order to block.
```

Comment:
Documentation does not seem to match the code. Blocking's first bool is only true if holding a shield and performing the necessary actions to block.